packages/hash/shared/src/entityStorePlugin.ts

```diff
@@ -29,31 +34,148 @@ type EntityStorePluginState = {
 
 type EntityStorePluginAction =
   | {
+      // @todo remove this
```

Comment:
It’d be useful to see more context in comments like this. Otherwise they may read as ‘remove this before PR is merged’

Reply:
Fair point – I'll expand. Need to make comments less of "notes for future nate" and more understandable by wider team.